src/AzureExtension/DataModel/DataObjects/Notification.cs

```diff
@@ -0,0 +1,262 @@
+
+    public string Title { get; set; } = string.Empty;
+
+    public string Identifier { get; set; } = string.Empty;
+
+    public string Result { get; set; } = string.Empty;
+
+    public string Description { get; set; } = string.Empty;
+
+    public string HtmlUrl { get; set; } = string.Empty;
+
+    public long ToastState { get; set; } = DataStore.NoForeignKey;
+
+    public long TimeCreated { get; set; } = DataStore.NoForeignKey;
+
+    [Write(false)]
+    private DataStore? DataStore { get; set; }
+
+    [Write(false)]
+    [Computed]
+    public DateTime CreatedAt => TimeCreated.ToDateTime();
+
+    [Write(false)]
+    [Computed]
+    public Project Project => Project.Get(DataStore, ProjectId);
+
+    [Write(false)]
+    [Computed]
+    public Repository Repository => Repository.Get(DataStore, RepositoryId);
+
+    [Write(false)]
+    [Computed]
+    public NotificationType Type => (NotificationType)TypeId;
+
+    [Write(false)]
+    [Computed]
+    public bool Toasted
+    {
+        get => ToastState != 0;
+        set
+        {
+            ToastState = value ? 1 : 0;
+            if (DataStore is not null)
+            {
+                try
+                {
+                    DataStore.Connection!.Update(this);
+                }
+                catch (Exception ex)
+                {
+                    // Catch errors so we do not throw for something like this. The local ToastState
+                    // will still be set even if the datastore update fails. This could result in a
+                    // toast later being shown twice, however, so report it as an error.
+                    Log.Error(ex, "Failed setting Notification ToastState for Notification Id = {Id}");
+                }
+            }
+        }
+    }
+
+    public override string ToString() => $"[{Type}][{Project}] {Title}";
+
+    /// <summary>
+    /// Shows a toast formatted based on this notification's NotificationType.
+    /// </summary>
+    /// <returns>True if a toast was shown.</returns>
+    public bool ShowToast()
+    {
+        if (Toasted)
+        {
+            return false;
+        }
+        else if (LocalSettings.ReadSettingAsync<string>("NotificationsEnabled").Result == "false")
+        {
+            Toasted = true;
+            return false;
+        }
+
+        return Type switch
+        {
+            NotificationType.PullRequestRejected => ShowPullRequestRejectedToast(),
+            NotificationType.PullRequestApproved => ShowPullRequestApprovedToast(),
+            _ => false,
+        };
+    }
+
+    private bool ShowPullRequestRejectedToast()
+    {
+        try
+        {
+            Log.Information($"Showing Notification for {this}");
+            var nb = new AppNotificationBuilder();
+            nb.SetDuration(AppNotificationDuration.Long);
+            nb.AddArgument("htmlurl", HtmlUrl);
+            nb.AddText($"❌ {Resources.GetResource("Notifications_Toast_PullRequestRejected/Title", Log)} - {Description}");
```

Comment:
Different cultures use different symbols.  I don't know how to show these differences.  Just a thought.

https://en.wikipedia.org/wiki/Check_mark#International_differences

Reply:
No idea, but if it is a localization issue it is also present in GitHub extension also (both use it) so if a change is required I'd prefer a targeted PR for that.